core/src/main/java/edu/ucr/cs/riple/core/Annotator.java

```diff
@@ -256,14 +257,27 @@ private void forceResolveRemainingErrors(
     // the method level.
     Set<AddAnnotation> nullUnMarkedAnnotations =
         remainingErrors.stream()
-            // filter non-method regions.
-            .filter(error -> !error.encMethod().equals("null"))
             // find the corresponding method nodes.
-            .map(error -> tree.findNode(error.encMethod(), error.encClass()))
-            // impossible, just sanity check or future nullness checker hints
+            .map(
+                error -> {
+                  if (!error.encMethod().equals("null")) {
+                    return tree.findNode(error.encMethod(), error.encClass());
+                  }
+                  if (error.nonnullTarget == null) {
+                    // Just a sanity check.
+                    return null;
+                  }
+                  if (error.messageType.equals("PASS_NULLABLE")) {
+                    OnMethod calledMethod = error.nonnullTarget.toMethod();
+                    return tree.findNode(calledMethod.method, calledMethod.clazz);
+                  }
+                  return null;
+                })
             .filter(Objects::nonNull)
             .map(node -> new AddMarkerAnnotation(node.location, config.nullUnMarkedAnnotation))
             .collect(Collectors.toSet());
+    // Collect methods received a nullable i their nonnull parameters where called in field
+    // declaration regions of classes (method == "null).
```

Comment:
Is this the right place for this comment?  I don't understand how it relates to the code

Reply:
953786d6bb704e12d25d60a1ac190f2c3a164737 sorry for that. Removed the comment. 953786d6bb704e12d25d60a1ac190f2c3a164737